Symfony/CS/Fixer/Contrib/PhpUnitStrictFixer.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+final class PhpUnitStrictFixer extends AbstractFixer
+{
+    static private $phpUnitMethods = array(
+        "assertAttributeEquals" => "assertAttributeSame",
+        "assertAttributeNotEquals" => "assertAttributeNotSame",
+        "assertEquals" => "assertSame",
+        "assertNotEquals" => "assertNotSame",
```

Comment:
It mimics strict fixer, where we handle both `===` and `!==`.
One may use strict comparision as rule or not.
In your case I simply say that then you don't follow that rule here - you `often` you sth. This rule is when you want `always` use sth